parser.py

```diff
@@ -1030,8 +1031,41 @@ def __init__(self, dir_archive, dir_output):
         self.files_input_tweets = find_files_input_tweets(self.dir_input_data)
 
 
+def is_archive(path):
+    """Return true if there is a Twitter archive at the given path"""
+    return os.path.isfile(os.path.join(path, 'data', 'account.js'))
+
+
+def find_archive():
+    """
+    Search for the archive
+    1. First try the working directory.
+    2. Then try the script directory.
+    3. Finally prompt the user.
+    """
+    if is_archive('.'):
+        return '.'
+    script_dir = os.path.dirname(__file__)
+    if script_dir != os.getcwd():
+        if is_archive(script_dir):
+            return script_dir
+    print('Archive not found in working directory or script directory.\n'
+          'Please enter the path of your Twitter archive, or just press Enter to exit.\n'
+          'On most operating systems, you can also try to drag and drop your archive folder '
+          'into the terminal window, and it will paste its path automatically.\n')
+    # Give the user as many attempts as they need.
+    while True:
+        input_path = input('Archive path: ')
+        if not input_path:
+            exit()
+        if is_archive(input_path):
+            return input_path
+        print(f'Archive not found at {input_path}')
+
+
 def main():
-    paths = PathConfig(dir_archive='.', dir_output='.')
+    archive_path = find_archive()
+    paths = PathConfig(dir_archive=archive_path, dir_output=archive_path)
```

Comment:
I would suggest to keep the `dir_output='.'` here.

That way, users can purposefully start the parser from a different location, and have the output not be written into the archive folder. 

Then they don't have to manually pluck it apart from the original archive files when moving the output somewhere else.

Reply:
I agree that it would be a nice feature if the user can choose where the output goes, and if that would allow the output to be outside the archive.

But the change that you suggest seems like a rather weird hack to me, and not obvious to the user how to use it to their advantage.

The PR #120 adds a (hard coded) output folder inside the archive, and will be merged soon.

For configurable output folders, I think we should have a separate issue. Or have a [discussion](https://github.com/timhutton/twitter-archive-parser/discussions) about configurable options in general.

Reply:
Good point. 

Also, I didn't think about what could happen this way: you could accidentally overwrite the output from a different archive.

So let's just keep writing into the archive folder for now and wait for #120 to make it easier to separate output from input.

Sorry, I obviously didn't think this through.